src/StatusQ/Components/StatusListItem.qml

```diff
@@ -73,24 +52,69 @@ Rectangle {
         height: 40
         isIdenticon: false
     }
-    property string label: ""
-
-    property int type: StatusListItem.Type.Primary
+    property StatusIconSettings titleIcon1: StatusIconSettings {
+        width: dummyImage.width
+        height: dummyImage.height
+        background: StatusIconBackgroundSettings {
+            width: 10
+            height: 10
+        }
+        // Only used to get implicit width and height from the actual image
+        property Image dummyImage: Image {
+            source: titleIcon1.name ? "../../assets/img/icons/" + titleIcon1.name + ".svg": ""
+            visible: false
+        }        
```

Comment:
I needed to use this in the StatusMessage header as well. that's because the image size is rather small and width and height are not the same, and because they are so tiny, setting a default one really breaks the way to image looks. Not sure if Noelia has a better way :)

Reply:
I agree with you that this is not the cleanest approach and, yes, we could approach it by using for example a fixed size of 6x8 (as in this case is the size of these tiny specific icons) but you lose some flexibility as if you use the component with bigger icon sizes, it will not be rendered correctly (or at least is what I’ve experimented). Maybe this shouldn't be happening, and we should review our base class `StatusRoundIcon`. 

- For example, if I fix it to 10x10:
![10x10](https://user-images.githubusercontent.com/97019400/150951916-17caf37a-0a62-4a31-8f0d-1fbce8a15b99.PNG)

- This is the dynamic size approach, in this case, same as 6x8:
![6x8](https://user-images.githubusercontent.com/97019400/150951858-ba3aa13f-777a-4601-aa5c-3d323905b534.PNG)

Can you appreciate the difference?

This approach is, as well, used in the `StatusMessageHeader.qml` (as @Khushboo-dev-cpp says) and gives some dynamic opportunities to the component.

Reply:
Wonderful, thanks for clarifying!